compile error: makedev moved to sys/sysmacros.h

Now that all warnings are treated as errors (since -Werror patch), the following warning breaks compilation on newer GNU C Library installations:

```
[  8%] Building CXX object media_driver/CMakeFiles/iHD_drv_video.dir/linux/common/os/libdrm/xf86drm.c.o
media_driver/linux/common/os/libdrm/xf86drm.c:454:13: error: In the GNU C Library, "makedev" is defined
 by <sys/sysmacros.h>. For historical compatibility, it is
 currently defined by <sys/types.h> as well, but we plan to
 remove this soon. To use "makedev", include <sys/sysmacros.h>
 directly. If you did not intend to use a system-defined macro
 "makedev", you should undefine it after including <sys/types.h>. [-Werror]
     return drmOpenDevice(makedev(DRM_MAJOR, minor), minor, type);
             ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~                                                                                                                                                                                                                                                                                                                         
cc1plus: all warnings being treated as errors
make[2]: *** [media_driver/CMakeFiles/iHD_drv_video.dir/build.make:8247: media_driver/CMakeFiles/iHD_drv_video.dir/linux/common/os/libdrm/xf86drm.c.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:164: media_driver/CMakeFiles/iHD_drv_video.dir/all] Error 2
make: *** [Makefile:152: all] Error 2
```

--- a/media_driver/linux/common/os/libdrm/xf86drm.c
+++ b/media_driver/linux/common/os/libdrm/xf86drm.c
@@ -47,6 +47,7 @@
 #include <signal.h>
 #include <time.h>
 #include <sys/types.h>
+#include <sys/sysmacros.h>
 #include <sys/stat.h>
 #define stat_t struct stat
 #include <sys/ioctl.h>

